migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -12,402 +12,158 @@ import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+Application downtime begins here.
 </Highlight>
 
-## 6. Migrate initial data from source to target
-
-### 6.a. Migrate table data from source to target
-
-Use pgcopydb and the snapshot ID obtained in [step 3][step-3]
-(/tmp/pgcopydb/snapshot) to copy the data. This action can take a significant
-amount of time, up to several days, depending on the size of the source
-database. You must wait until it finishes before moving on to the next step. To
-start copying data, execute the following command:
-
-```sh
-pgcopydb copy table-data \
-  --source "$SOURCE" \
-  --target "$TARGET" \
-  --snapshot $(cat /tmp/pgcopydb/snapshot) \
-  --split-tables-larger-than '10 GB'  \
-  --table-jobs 8
-```
-
-- `--snapshot` specifies the synchronized [snapshot][snapshot] to use when
-  retrieving the data. It must be set up to the snapshot exported in
-  [step 3][step-3], that guarantees that the only data copied is the one that
-  existed on the database prior to the creation of the replication slot.
-
-- `--split-tables-larger-than` defines the table size threshold after which
-  more than once process is going be use at the same time to process a single
-  source table.
-
-- `--table-jobs` is the number of concurrent `COPY` jobs to run.
-
-### 6b. Dump the database indexes, constraints and other objects
-
-<DumpPostDataSourceSchema />
-
-### 6c. Load the indexes, constraints and other objects into the target database
-
-```sh
-psql -X -d "$TARGET" \
-  -v ON_ERROR_STOP=1 \
-  --echo-errors \
-  -f post-data-dump.sql
-```
-
-## 7. Apply the replicated changes from source
-
-With the schema and initial data loaded, you can now apply the buffered and
-live changes emitted by the replication slot:
-
-```sh
-pgcopydb stream sentinel set apply --source $SOURCE
-```
-
-The replication progress can be monitored by querying the [pg_stat_replication
-view][pg-stat-replication-docs]:
-
-```sh
-psql $SOURCE \
-  -f - <<'EOF'
-SELECT replay_lag
-FROM pg_stat_replication
-WHERE application_name='pgcopydb' AND state='streaming'
-EOF
-```
+When you’re _ready to take downtime_ (i.e., stop normal operations) on your
+production applications that are dependent on your source database and the
+_lag between the databases is below 30 megabytes_, you should stop your
+applications that perform DML queries on your source database. This is
+the downtime phase and will last until you have completed the validation
+step (4). Be sure to go through the validation step (4) and prepare the
+validation queries before you enter the downtime phase to keep the overall
+downtime minimal.
 
-- `replay_lag`: Time elapsed between flushing recent WAL locally and receiving
-  notification that this standby server has written, flushed and applied it.
+Stopping DML queries on your source database allows the live migration
+image to make the target database even with the source. This will be
+evident when the lag outputs 0 megabytes.
```

Comment:
```suggestion
Stopping writes to the source database allows the live migration process to
finish replicating data to the target database. This will be evident when the
replication lag reduces to 0 megabytes.
```